src/osd/OSD.h

```diff
@@ -685,6 +685,21 @@ class OSDService {
     return true;
   }
 
+  void dumps_scrub(Formatter *f) {
+    assert(f != nullptr);
+    Mutex::Locker l(sched_scrub_lock);
+
+    f->open_array_section("sched scrubs");
```

Comment:
an XML tag cannot contain space in it. see https://www.w3.org/TR/2008/REC-xml-20081126/#NT-NameChar.

so probably we can simply put "scrubs" as the tag name?